frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -97,11 +97,19 @@ class ListDropdown_ extends React.Component {
 
       const keyChanged = currentState.selectedKey !== nextProps.selectedKey;
       const keyKindChanged = currentState.selectedKeyKind !== nextProps.selectedKeyKind;
-      const selectedKey =
+      let selectedKey =
         keyChanged || keyKindChanged
           ? getKey(nextProps.selectedKey, nextProps.selectedKeyKind)
           : currentState.selectedKey;
 
+      if (selectFirst) {
```

Comment:
selectFirst is string even if I say "No" then also the first will be selected.

Reply:
What will happen if we have default snapshot class and also `selectFirst` set to true. Which will get more precedence?